scrapers/FilthyFamily.yml

```diff
@@ -3,30 +3,37 @@ sceneByURL:
   - action: scrapeXPath
     url:
       - filthyfamily.com
```

Comment:
this should removed? (to avoid users trying to scrape using the now paywalled urls )

Reply:
I left this here as I guessed that if you have access to the site, you could (as a logged in member of the site), scrape the site pages.

I could probably put some comments in this scraper file to make that more obvious, and that non-members should use the mobile domain.

Reply:
or....

I could see if the URL regex replacer thing would work to transform the main domain to the subdomain, something like:

```
    queryURL: "{url}"
    queryURLReplace:
      url:
        - regex: '(?<!mobile\.)filthyfamily\.com'
          with: mobile.filthyfamily.com
```

Reply:
yeah i thought of that as well
it should work provided that all urls translate like you mentioned in your first post

Reply:
yeah it works like:
```
sceneByURL:
  - action: scrapeXPath
    url:
      - filthyfamily.com
      - mobile.filthyfamily.com
    scraper: sceneScraper
    queryURL: "{url}"
    queryURLReplace:
      url:
        - regex: https://filthyfamily\.com
          with: https://mobile.filthyfamily.com
```